### ISSUE TITLE
SovrenClient.DeleteMultipleDocuments throws System.NotSupportedException: 'DELETE cannot have a body'

### ISSUE THREAD

**Issue by leorg99** [textkernel/tx-dotnet#42]
Using Sovren.SDK  v1.5.1 and .NET 5.0 and getting this exception:

```
{
	"ClassName": "System.NotSupportedException",
	"Message": "DELETE cannot have a body",
	"Data": {},
	"InnerException": null,
	"HelpURL": null,
	"StackTraceString": "   at Sovren.Rest.RestRequest.AddJsonBody(String json, Encoding encoding)",
	"RemoteStackTraceString": null,
	"RemoteStackIndex": 0,
	"ExceptionMethod": null,
	"HResult": -2146233067,
	"Source": "Sovren.SDK",
	"WatsonBuckets": null
}
```

**Comment by sovren-jwesson** [comment]
@LeorGreenberger thanks for reporting this. I'll get this fixed today.

**Comment by leorg99** [comment]
@sovren-jwesson  Thanks for the quick fix! I see v1.5.2 was released but doesn't look like it made it to nuget? https://www.nuget.org/packages/Sovren.SDK/

**Comment by sovren-jwesson** [comment]
@LeorGreenberger yes, I forgot to update the project version before pushing to nuget so that didn't work. That will be available shortly, but it will be 1.5.3. I'll respond here once it's available.

**Comment by sovren-jwesson** [comment]
@LeorGreenberger v1.5.3 is available on nuget now. Sorry about that, the nuget publish action failed silently because of an option I was setting, so I was just waiting for the package to show up on nuget before reaching back out. That is now fixed (#44), so if the project version doesn't get updated the nuget push will fail and I will know about it immediately. Let us know if you run into any more issues. Thanks!

**Comment by leorg99** [comment]
@sovren-jwesson Thank you very much!!
